packages/sso-button-react/src/SsoButton/SsoButton.tsx

```diff
@@ -28,6 +28,7 @@ export type SsoButtonProps = {
     button?: string
   }
   logoutTooltip?: string
+  disableAction?: boolean
```

Comment:
only applied when `onLogout` or `onLogin` is passed. However, making this prop available only under one of these conditions is a bit verbose.